client/src/pages/Register/Login/index.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import {
+  TextField, Button, Alert, FormHelperText,
+} from '@mui/material';
+import axios from 'axios';
+import './style.css';
+
+const Login = () => {
+  const [value, setValue] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState('');
+  const history = useHistory();
+  const sendRequest = async (e) => {
+    e.preventDefault();
+    try {
+      if (value.email && value.password) {
+        if (value.password.length >= 7) {
+          await axios.post('api/v1/login', value);
+          history.push('/');
```

Comment:
```suggestion
          history.push(`/organization/${id}/${name}`);
```
When the user login to his account he has to be redirected to his profile.